.travis.yml

```diff
@@ -82,8 +82,8 @@ jobs:
       script:
         - yarn build
         - pip install --user awscli
-        - aws s3 cp ./dist s3://$AWS_ASSETS_BUCKET/$TRAVIS_COMMIT --recursive --acl public-read
-        - ./scripts/comment-preview-link
+        - yarn add @typeform/jarvis
+        - DEBUG=jarvis yarn run jarvis deploy --path dist --preview --notify-preview
```

Comment:
just a side note - we also have a lib folder generated along with the dist. We don't need to upload it anywhere, do we?

Reply:
I think that is used when you install the lib via npm. We are not deploying it right now and everything works :)

Reply:
@mathio cool! Mentioned that just in case :)

Reply:
Yes, we should always point out if anything seems odd. More 👀  = higher code quality.